src/core/js/views/drawerView.js

```diff
@@ -86,7 +86,14 @@ define([
             this._isCustomViewVisible = true;
             Adapt.trigger('drawer:empty');
             this.showDrawer();
+
             this.$('.drawer-holder').html(view);
+
+            var model = this.collection.findWhere({
```

Comment:
sure , i have added the title in PLP object in course.json 
```
 "_pageLevelProgress": {
        "_isEnabled": true,
        "title": "page Level Progress",
        "_showPageCompletion": false,
        "_altText": "of components completed on this page."
    },  ``` 
but unable to see the title for PLP drawer like resources